### DISCUSSION TITLE
[@types/custom-functions-runtime] overload signature is not compatible with its implementation signature

### DISCUSSION THREAD

**Discussion by shaun-pendrigh** [DefinitelyTyped/DefinitelyTyped#60942] [Issues with a @types package] [answered]
Not sure what I am missing here. Using the Office custom-functions-runtime type definitions in my Add-In project. In order to write tests I need to create a dummy implementation of this type in my project (plan to mock later but could not get the syntax correct). In creating this dummy implementation I came across a conflict in that the CustomFunctions.associate method has two overloads and their parameters are in conflict. I am new to Typescript but reading the documentation https://www.typescriptlang.org/docs/handbook/2/functions.html#function-overloads it seems the overloads are required to be compatible. So I am not sure what the correct way would be to implement a dummy version of this type in my project. Any guidance would be appreciated, thanks.

[a comment by an automated account is not shown]

**Comment by ElizabethSamuel-MSFT** [comment]
Pinging @alison-mk

**Comment by shaun-pendrigh** [comment, marked as answer]
This can be closed thanks, was my mistake
